query/autocomplete_defaults.js

```diff
@@ -60,8 +60,10 @@ module.exports = _.merge({}, peliasQuery.defaults, {
   // generic multi_match config
   'multi_match:type': 'cross_fields',
   'multi_match:ngrams_strict:type': 'phrase',
+  'multi_match:ngrams_fuzzy:operator': 'and',
```

Comment:
I'm a bit confused as to where this is used 🤔 
Same goes for `multi_match:first_tokens_only_fuzzy:operator`

Reply:
This is used by `pelias-query` to construct the `multi_match:ngrams_fuzzy` view later on, which gets to look like this:
```            'multi_match': {
              'fields': ['name.default', 'name.en'],
              'analyzer': 'peliasQuery',
              'query': 'test',
              'boost': 100,
              'type': 'best_fields',
              'operator': 'and',
              'fuzziness': 1,
              'max_expansions': 40,
              'prefix_length': 1
            }
```
The `and` operator for a `match` query basically means that all words from the query text must be contained in the document field - otherwise the default would be `or` - any word from the query text contained in the document field would do. (https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html - `operator` section)